standards/document-the-security-vulnerability-policy.md

```diff
@@ -0,0 +1,28 @@
+---
+category: Operating services
+---
+# Document the security vulnerability policy for your service
+
+All MOJ production (live) domains **must** use security.txt in order to document the security vulnerability policy (including contact methods and so on) for your service.
+
+This includes where the domain is primarily operated by a partner (for example, commercial supplier) but the MOJ remains primarily responsible for cyber security of the domain and underlying websites or digital services.
+
+Using security.txt does not preclude you from documenting in other ways, but at minimum security.txt *has* to be used.
+
+## Deploy the HTTP 301 redirect
+
+Follow the [guidance from the Cyber Security team](https://ministryofjustice.github.io/security-guidance/contact/implement-security-txt/#implementing-securitytxt){:target="_blank"}.
+
+### What is security.txt
```

Comment:
You can kill this section in favour of just linking the first reference to `security.txt` to https://securitytxt.org/.